WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewModel.swift

```diff
@@ -50,6 +55,7 @@ final class ProductFormViewModel: ProductFormViewModelProtocol {
     private let productNameSubject: PassthroughSubject<String, Never> = PassthroughSubject<String, Never>()
     private let isUpdateEnabledSubject: PassthroughSubject<Bool, Never> = PassthroughSubject<Bool, Never>()
     private let newVariationsPriceSubject = PassthroughSubject<Void, Never>()
+    private let isEligibleForBlazeSubject = PassthroughSubject<Void, Never>()
```

Comment:
Nit: let's rename this to avoid confusion with `isEligibleForBlaze` below:

```suggestion
    private let blazeEligiblityUpdateSubject<Void, Never>()
```